### PR TITLE
Try/standard version

### PR DESCRIPTION
**Pull request by zeirash**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -155,6 +155,17 @@ jobs:
...
           yarn typecheck
 
+  versioning:
+    docker:
+      - image: node:12
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: |
+          npm i -g standard-version
```
[the remark this answers is not in the document]
Comment (chrsep) [reply .circleci/config.yml]: I also don't think we need npm publish, that's for publishing a new package right?

[the remark this answers is not in the document]
Comment (zeirash) [reply .circleci/config.yml]: hmm i see

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,963 @@
+# Changelog
```
[the remark this answers is not in the document]
Comment (zeirash) [reply CHANGELOG.md]: we need to delete this file first?

File: .circleci/config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (zeirash) [reply .circleci/config.yml]: should i need go to frontend folder first to use yarn command?

[the remark this answers is not in the document]
Comment (zeirash) [reply .circleci/config.yml]: should i need to go to frontend folder to use yarn command?

[the remark this answers is not in the document]
Comment (chrsep) [reply .circleci/config.yml]: > should i need go to frontend folder first to use yarn command?

Nah, you can run it everywhere.

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chrsep) [reply CHANGELOG.md]: > we need to delete this file first?

Hmmm no need i think. We’ll just need to edit it, removing duplicate and useless logs.

File: .circleci/config.yml [hunk 3]
```diff
@@ -155,6 +155,19 @@ jobs:
...
+    docker:
+      - image: node:12
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: |
+          yarn global add standard-version
+          standard-version release --no-verify
+      - run: |
+          yarn global add git
```
Comment (chrsep) [inline .circleci/config.yml]: I think we should test this new config first. Its fine to remove the filter for the versioning job and allow it to run on this branch first. Also, it should push to this branch too.

Here's a [guide](https://support.circleci.com/hc/en-us/articles/360018860473-How-to-push-a-commit-back-to-the-same-repository-as-part-of-the-CircleCI-job) to push back to GitHub. 

I've just added a user key to do the push. Just need to add the git email and name and then do the push.

You can also try run the job locally using the [circleci cli](https://circleci.com/docs/2.0/local-cli/) if you want.

File: .versionrc [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "bumpFiles": [
+        {
+            "filename": "VERSION.txt",
```
Comment (chrsep) [inline .versionrc]: Just a thought, I don't think we need the `.txt` extension. Just `VERSION` is fine. Feels cleaner for me that way for some reason 🤓.

File: .circleci/config.yml (hunk 3 above)
Comment (zeirash) [reply .circleci/config.yml:168]: which git email and name should we use?

File: .versionrc (hunk 4 above)
Comment (zeirash) [reply .versionrc:4]: got it

File: .circleci/config.yml (hunk 3 above)
Comment (chrsep) [reply .circleci/config.yml:168]: Right hmmm I think we could use my email, chrsep@protonmail.com, Chrisando Pramudhita.

**Pull request merged**